src/legacy/ui/ui_exports/ui_export_defaults.js

```diff
@@ -51,7 +51,7 @@ export const UI_EXPORT_DEFAULTS = {
       'ui/field_editor/components/field_format_editor/register'
     ],
     visEditorTypes: [
-      'ui/vis/editors/default/default',
+      'ui/vis/editors/default/default_editor',
```

Comment:
Hey @timroes @spalger @flash1293 
I'm not sure about this change specifically.
Could please someone of you make sure this is right one. 
Thanks!